bin/install

```diff
@@ -278,10 +397,23 @@ configure_selected_shells() {
     else
       log_info "info" "configuring $shell in $dotfile_path"
       echo "$config_line" >> "$dotfile_path"
+      set_g_alias $dotfile_path
     fi
   done
 }
 
+#
+# set alias in dotfile
+#
+
+set_g_alias() {
+  if [ "$G_ALIAS_CHANGED" = true ]; then
+    log_info "info" "configuring $G_ALIAS as alias in $1"
```

Comment:
Let's remove this log since it would output "configuring X in file Y" twice for each shell. Since the setup of env vars and the setup of the aliases affect the same files, let's treat them as the same operation.